pubsub/sample.rb

```diff
@@ -106,3 +106,89 @@ def list_subscriptions
   end
 end
 # [END list_subscriptions]
+
+# [START print_topic_policy]
```

Comment:
Please put doc region tags inside methods so the code sample shows the relevant code (not wrapped in a method)